client/python/README.md

```diff
@@ -0,0 +1,105 @@
+# `gradio_client`: Use any Gradio app as an API -- in 3 lines of Python
+
+This directory contains the source code for `gradio_client`, a lightweight Python library that makes it very easy to use any Gradio app as an API. This library is CURRENTLY IN ALPHA. APIS MAY CHANGE. WE DO NOT RECOMMEND USING IT EXCEPT IN EXPERIMENTAL PROJECTS. 
```

Comment:
can we remove this comment? I think it discourages users unnecessarily - if this is trusted enough to be used for the core gradio library, then its good enough for our users, they'll open issues if things break

Reply:
I'll soften the message, but the library is not meant for external release. We can always change the names of variables, etc. for internal use but if people build on top of it, then it may cause their code to break.